### PR TITLE
JOSS submission changes 

### PR DESCRIPTION
**Pull request by kpolimis**
- added Quinlan 1993 citation
- discussed plotting with standard deviation instead of variance
- syntax changes

### CODE REVIEW

File: paper/paper.bib [hunk 1]
```diff
@@ -10,7 +10,8 @@ @article{wager_confidence_2014
 	month = jan,
 	year = {2014},
 	keywords = {bagging, jackknife methods, Monte Carlo noise, variance estimation},
-	pages = {1625--1651}
+	pages = {1625--1651},
+	file = {Wageretal2014.pdf:C\:\\Users\\Kivan\\Zotero\\storage\\7ZBXYKH2\\Wageretal2014.pdf:application/pdf}
```
Comment (arokem) [inline paper/paper.bib]: Please remove the line pointing to the file on your hard-drive

File: paper/paper.bib [hunk 2]
```diff
@@ -23,10 +24,16 @@ @misc{wager_randomforestci_2016
...
+	series = {{ICML}'93},
+	title = {Combining {Instance}-based and {Model}-based {Learning}},
+	isbn = {1-55860-307-7},
+	url = {http://dl.acm.org/citation.cfm?id=3091529.3091560},
+	booktitle = {Proceedings of the {Tenth} {International} {Conference} on {International} {Conference} on {Machine} {Learning}},
+	publisher = {Morgan Kaufmann Publishers Inc.},
+	author = {Quinlan, J. Ross},
+	year = {1993},
+	pages = {236--243},
+	file = {Quinlan1993.pdf:C\:\\Users\\Kivan\\Zotero\\storage\\XNZAZBY7\\Quinlan1993.pdf:application/pdf}
```
Comment (arokem) [inline paper/paper.bib]: Here as well

**Review by arokem — CHANGES_REQUESTED** [review]
Looks good. There's a bit of cruft in here pointing to local paths on your machine. Please remove that, and I will merge.

**Comment by kpolimis** [comment]
removed info related to my local machine and updated the paper text and README

**Pull request merged**